### ISSUE TITLE
Ability to run theories wihout specifying the Theories runner

### ISSUE THREAD

**Issue by TomDmitriev** [pholser/junit-quickcheck#47]
When I already have a junit runner I can't add @RunWith(Theories.class) to my tests. It would be nice if I could create and run theories without the Theories runner, using a junit rule for example.

**Comment by pholser** [comment]
That sounds like a great idea. Currently, junit-quickcheck leverages the theories machinery, which all is triggered by a specific runner. To disengage from that machinery and come up with something that can be used with any JUnit runner would be a lot of work, but might encourage wider adoption of property-based testing also.

https://github.com/pholser/junit-theories/issues/3

**Comment by pholser** [comment]
Closing - this is probably better handled as an enhancement to junit-theories, rather than to junit-quickcheck per se.

**Comment by runeflobakk** [comment]
Can I ask to reopen this issue? I would very much like to use junit-quickcheck with other testrunners. This could be realized using a [JUnit @Rule](https://github.com/junit-team/junit4/wiki/Rules) I assume.

E.g. with Mockito it was previously common to use [@RunWith(MockitoJUnitRunner.class)](https://github.com/mockito/mockito/blob/master/src/main/java/org/mockito/junit/MockitoJUnitRunner.java), but its use has been replaced by the more flexible [MockitoJUnit.rule()](https://github.com/mockito/mockito/blob/master/src/main/java/org/mockito/junit/MockitoRule.java), as it allows the "mockito behavior" to happen with various test runners.

**Comment by pholser** [comment]
@runeflobakk Certainly. I'm not sure how I'd go about this, but I'll study the Mockito rule to get a sense of what it's doing.
